src/config/constants.ts

```diff
@@ -99,6 +99,7 @@ export const TWITTER_URL = 'https://twitter.com/safe'
 
 // Legal
 export const IS_OFFICIAL_HOST = process.env.NEXT_PUBLIC_IS_OFFICIAL_HOST === 'true'
+export const OFFICIAL_HOSTS = /app\.safe\.global|.+\.5afe\.dev|localhost:3000/
```

Comment:
Todo: move this to GitHub Secrets so that it's a bit more difficult to overwrite.